packages/react-reconciler/src/ReactEventPriorities.new.js

```diff
@@ -7,9 +7,66 @@
  * @flow
  */
 
-export {
-  SyncLane as DiscreteEventPriority,
-  InputContinuousLane as ContinuousEventPriority,
-  DefaultLane as DefaultEventPriority,
-  IdleLane as IdleEventPriority,
+import type {Lane, Lanes} from './ReactFiberLane.new';
+
+import {
+  NoLane,
+  SyncLane,
+  InputContinuousLane,
+  DefaultLane,
+  IdleLane,
+  getHighestPriorityLane,
 } from './ReactFiberLane.new';
+
+export opaque type EventPriority = Lane;
+
+export const DiscreteEventPriority: EventPriority = SyncLane;
+export const ContinuousEventPriority: EventPriority = InputContinuousLane;
+export const DefaultEventPriority: EventPriority = DefaultLane;
+export const IdleEventPriority: EventPriority = IdleLane;
+
+let currentUpdatePriority: EventPriority = NoLane;
+
+export function getCurrentUpdatePriority(): EventPriority {
```

Comment:
Re: naming, I was thinking I would rename this to `getCurrentEventPriority`, and then rename the host config one to something like `getCurrentHostEventPriority`. Gonna leave as-is in this PR, though.

Reply:
Why bother to rename more than this if we're going to remove it?

Reply:
This is staying. It tracks a Lane now instead of a LanePriority.